src/main/java/com/google/sps/tool/EntityTestingTool.java

```diff
@@ -0,0 +1,59 @@
+package com.google.sps.tool;
+
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+
+import java.util.List;
+import java.util.ArrayList;
+import com.google.sps.data.Card;
+import com.google.sps.data.Folder;
+
+public class EntityTestingTool {
+
+  public static boolean checkForNoNullValues(Entity card) {
+    return (card.getProperty("blobKey") != null  &&
+      card.getProperty("labels") != null &&
+      card.getProperty("textTranslated") != null &&
+      card.getProperty("textNotTranslated") != null &&
+      card.getProperty("fromLang") != null &&
+      card.getProperty("toLang") != null &&
+      card.getProperty("cardKey") != null);
+  }
+
+  public static List<Folder> populateDatastoreWithFolders(Folder folderA, Folder folderB, DatastoreService datastore, String userKey) {
+    
+    Entity folderEntityA = folderA.createEntity(KeyFactory.stringToKey(userKey));
+    Entity folderEntityB = folderB.createEntity(KeyFactory.stringToKey(userKey));
+
+    // Update entity in datastore 
+    datastore.put(folderEntityA);
+    datastore.put(folderEntityB);
+
+    List<Folder> folders = new ArrayList<>();
+    folders.add(new Folder(folderEntityA));
+    folders.add(new Folder(folderEntityB));
+
+    return folders;
+  }
```

Comment:
I feel like more space would be conserved if you did one folder at a time. In your test you could just call this function twice. Also it would make the code more general because what if you had a test that needed three folders.

Reply:
+1. You could also consider [varargs](https://docs.oracle.com/javase/8/docs/technotes/guides/language/varargs.html), but I think calling the method multiple times is simplest.